### PR TITLE
OPENEUROPA-2165: [media_avportal] Missing information for some videos.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

Some videos have a different response format, thus the current code cannot retrieve the correct metadata from them. Quite a few examples were found by AMPATZIS Ioannis:

no thumbnail: https://audiovisual.ec.europa.eu/en/video/I-129872 https://audiovisual.ec.europa.eu/en/video/I-098040
no title: https://audiovisual.ec.europa.eu/en/video/I-053547 https://audiovisual.ec.europa.eu/en/video/I-001464
no thumbnail AND title: https://audiovisual.ec.europa.eu/en/video/I-056847
The reason seems a different format for the json response. Scope of the ticket is to cover these cases.

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

**Review by imanoleguskiza — DISMISSED** [review]

**Review by imanoleguskiza — DISMISSED** [review]

File: tests/src/FunctionalJavascript/MediaAvPortalCreateContentTest.php [hunk 1]
```diff
@@ -277,4 +293,42 @@ public function testAvPortalPhotoMediaEntityResponsive(): void {
     $this->assertContains('P038924-352937.jpg', $image_url);
   }
 
+  /**
+   * Fixtures of data for testing rendering of creating and rendering of.
+   */
+  protected function getVideoFixtures(): array {
+    return [
+      [
```
Comment (brummbar) [inline tests/src/FunctionalJavascript/MediaAvPortalCreateContentTest.php]: Let's add the key here so we can specify what we are testing, like "Video with no title"

File: src/AvPortalResource.php [hunk 2]
```diff
@@ -100,20 +104,52 @@ public function getThumbnailUrl(): ?string {
...
+   * Returns the thumbnail of a video resource.
+   *
+   * @return string|null
+   *   URL of the thumbnail.
+   */
+  protected function getVideoThumbnailUrl(): ?string {
+    $first_media_json = reset($this->data['media_json']);
+
+    // If available, get a language agnostic thumbnail.
+    if (isset($first_media_json['INT']['THUMB'])) {
```
Comment (brummbar) [inline src/AvPortalResource.php]: Let's add an extra fallback here. After checking if the international version is available, check for the current site language. Then return the first available in the languages array.

**Review by brummbar — CHANGES_REQUESTED** [review]
Let's add a service that allows to refresh existing avportal media entities so websites can refresh their data if needed.

**Review by hernani — APPROVED** [review]
Approved.

**Review by brummbar — APPROVED** [review]

**Pull request merged**